HISTORY.rst

```diff
@@ -50,6 +50,8 @@ Unreleased Changes (3.10)
     * Updating the ``taskgraph`` requirement to ``0.11.0`` to resolve an issue
       where modifying a file within a roughly 2-second window would fool
       ``taskgraph`` into believing that the file had not been modified.
+    * Fixed a bug where some input rasters with NaN nodata values would cause
+      bad behavior.
```

Comment:
How about "... would go undetected as nodata and yield unexpected results"?

Reply:
That's much better. https://github.com/natcap/invest/pull/770/commits/0fe1c064074aa824064cc10a761c5bcf6f518a76